src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1323,16 +1326,19 @@ public void statusUpdatedDateRemainsAfterUpdate() throws Exception {
   }
 
   @Test
-  public void statusUpdatedDateIsNullOnSubsequentUpdates() throws Exception {
+  public void statusUpdatedDateIsUnchangedOnSubsequentUpdates() throws Exception {
```

Comment:
Thank you for changing the name of this test to reflect the changes

Reply:
Does this test check that status date does not change when the item is updated and status hasn't been changed?

Reply:
That's what it looks like.  I'm not super-familiar with the testing syntax, but it looks like what it's doing is copying the item data and then overwriting it to force an update, then checks to see if the status data has changed.  Hmm.  It might then be better if I only grab the items created date once, and use that for all the tests.  I'll make that change.

Reply:
I wonder if it is worth making that aspect explicit in the name

```suggestion
  public void statusUpdatedDateIsUnchangedAfterAnUpdateThatDoNotChangeStatus() throws Exception {
```